testing/openjdk12/APKBUILD

```diff
@@ -0,0 +1,307 @@
+# Contributor: Simon Frankenberger <simon-alpine@fraho.eu>
+# Maintainer: Simon Frankenberger <simon-alpine@fraho.eu>
+pkgname=openjdk12
+pkgver=12.0.2_p10
+_pkgver=${pkgver/_p/+}
+pkgrel=0
+pkgdesc="Oracle OpenJDK 12"
+url="https://hg.openjdk.java.net/jdk-updates/jdk12u"
+arch="all !x86 !armhf !armv7" # openjdk11 is not yet available on 32 bit arches
```

Comment:
How does openjdk11 relate?

Reply:
The openjdk builds depend on the previous JDK. As OpenJDK11 is only available for 64 bit arches this blocks this aport